src/Command/SyncUserCommand.php

```diff
@@ -71,7 +71,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 'Display Name',
                 'Pronouns',
                 'Email',
-                'Phone Number'
+                'Phone Number',
```

Comment:
Is adding a trailing comma just a PHP style thing that is the convention now?

Reply:
We haven't done lint enforcement yet so it's not convention today. I was messing with this array and added the trailing comma, I left it in because I like how always having a last item comma makes git diffs easier to read because it doesn't indicate that the comma is a change to the line.